rm links to auth samples /2 (#32263)
dotnet/AspNetCore.Docs: aspnetcore/security/authentication/samples.md

Reviewer: This page no longer seems useful, I think we would do better to delete it, and redirect to the overview.

Author: I knew you'd know what to do. thanks

---
title: Authentication links ASP.NET Core
author: rick-anderson

> description: Provides links to the authentication topics in the ASP.NET Core repository.

Reviewer: ```suggestion
description: Provides links to the authentication articles in the ASP.NET Core repository.
```

By [Rick Anderson](https://twitter.com/RickAndMSFT)

> See  <xref:security/authentication/index> and the topics in the under Authentication for and overview of authentication and code snippets.

Reviewer 2: NIT stray space and perhaps bold for the ToC heading ...

```suggestion
See <xref:security/authentication/index> and the topics in the under **Authentication** for and overview of authentication and code snippets.
```

> See <xref:security/authentication/index> and the topics in the under **Authentication** for an overview of authentication and code snippets.

Reviewer: ```suggestion
See <xref:security/authentication/index> and the articles in the table of contents under **Authentication** for an overview of authentication and code snippets.
```

I'm guessing the TOC is what you had in mind.
